dataland-frontend/src/components/wrapper/AuthorizationWrapper.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <div v-if="hasUserRequiredRole">
+  <div v-if="hasUserRequiredRole || isUserDataOwner">
     <slot></slot>
```

Comment:
I am certified data owner for the company Abernathy.
Now If I visit the edit page for some dataset (see URL) of Abernathy, I get this:

![image](https://github.com/d-fine/Dataland/assets/99255296/157bb02a-4622-4e6f-95fa-bc4978f33f82)

Reply:
Should work now

Reply:
Seite funktioniert jetzt, ja.

Aber:  Wenn ich das dataset anschaue und dann auf "Edit" klicke, dann sehe ich kurz "you do not have permission".
Nach einem Augenblick verschwindet das jedoch und ich sehe die upload page.

Reply:
Aber es müsste doch ein Loading spinner zu sehen sein, der dir sagt, dass gerade nach data-ownership gesucht wird?

Reply:
Ich habe das v-else jetzt in eine v-if geändert, welches waitingForDataOwnershipData testet und nur wenn das auf false steht und man keine uploader rechte hat, sieht man die "Permission denied page". 
Was meinst du? Ist das besser so?

Reply:
Ich muss mir den Code mal anschauen. Bisher hab ich nur die UI händisch getestet, und lokal war das Problem gerade bei mir immer noch vorhanden.

Reply:
In pair programming gefixt